doc/src/proxies.md

```diff
@@ -0,0 +1,23 @@
+# Proxies
+
+`rustup` provides proxy wrapper through [components](https://github.com/rust-lang/rustup/blob/master/doc/src/concepts/components.md) and you can see the list of the proxies [here](https://github.com/rust-lang/rustup/blob/22dc71adbd7e19de0090f991b3609e5f14910dcb/src/lib.rs#L17-L28)
+
+- `rustc`: `rustc` is a proxy for the compiler for the Rust programming language, provided by the project itself and it matches `rustc` component.
```

Comment:
I think, given this is explicitly already a list of proxies, we can simplify the language here a little:

```suggestion
- `rustc` is the compiler for the Rust programming language, provided by the project itself and comes from the `rustc` component.
```

Note how I don't say "proxy" and I don't bother repeating the name of the proxy either, there's simply no need.

Reply:
You can make similar changes to all the lines.  No need to keep saying "proxy"